### PR TITLE
completed sequelize product model

### PR DESCRIPTION
**Pull request by nrod80**
#3 merge once approved by both of you

### CODE REVIEW

**Comment by clocasto** [comment]
The get methods for price/MSRP should return value / 100 instead of value \* 100. Also, could we DRY those two sets of methods up?

File: server/db/models/product.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+'use strict';
+var Sequelize = require('sequelize');
+var db = require('../_db');
+
+module.exports = db.define('product', {
+    name: {
+        type: Sequelize.STRING,
+        allowNull: false
+    },
+    manufacturer: {
```
Comment (zekenie) [inline server/db/models/product.js:10]: could be its own table. probably fine for now

File: server/db/models/product.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        type: Sequelize.STRING,
+        allowNull: false,
+    },
+    msrp: {
```
Comment (zekenie) [inline server/db/models/product.js:14]: what's this?

Comment (nrod80) [reply server/db/models/product.js:14]: the MSRP column corresponds to the official list price of a product, whereas the price column is the price at which our website would sell the product - basically our margins.

Comment (zekenie) [reply server/db/models/product.js:14]: Great thanks

File: server/db/models/product.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    description: {
+        type: Sequelize.TEXT,
+        allowNull: false
+    },
+    imageUrls: {
+        type: Sequelize.ARRAY(Sequelize.STRING),
+        allowNull: false
+    },
+    type: {
+        type: Sequelize.ENUM('base', 'ram', 'hdd', 'cpu'),
```
Comment (curiousYi) [inline server/db/models/product.js]: Should we also have GPU as a type as well?

**Comment by curiousYi** [comment]
Looks good to me.

**Pull request merged**